### PR TITLE
chore: release 0.2.0

### PR DESCRIPTION
**Pull request by miraclx**
First pre-release of `cargo-near`

Also removed the author line from the help output, matching `neard`'s format.

### CODE REVIEW

**Review by austinabell — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Build files
 /target
-Cargo.lock
```
Comment (itegulov) [inline .gitignore:3]: You are probably already aware of this, but `cargo install` ignores lockfile when executed without `--locked`. Nothing actionable here, just thought I would note.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,53 @@
+<!-- markdownlint-disable MD041 -->
```
Comment (itegulov) [inline README.md:1]: Why is this necessary? Don't we have a top-level heading in the first line?

**Review by itegulov — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (miraclx) [reply .gitignore:3]: Yeah, that inclusion is for git.

File: README.md (hunk 2 above)
Comment (miraclx) [reply README.md:1]: typo. `MD041` -> `MD014`
